script/generate.rb

```diff
@@ -12,3 +13,14 @@
   html.gsub!("title: $TITLE", "title: #{f.name}")
   IO.write("formula/#{f.name}.html", html)
 end
+
+Cask::Cask.each do |c|
+  json_filename = "#{File.basename(c.sourcefile_path,'.rb')}.json"
+  IO.write("_data/cask/#{json_filename}", JSON.pretty_generate(c.to_h))
+  FileUtils.cp "_api_formula.json.in", "api/cask/#{json_filename}"
```

Comment:
Does this file work perfectly? If so, can it be renamed to `_api.json.in` instead? Thanks!

Reply:
Yes it does work (see below). Should this also be adjusted for formula?

```
christianbernard@Dirac-MBP formulae.brew.sh (feature/formulae.brew.sh-generate-cask-json) $ cat _data/cask/chromium.json
{
  "name": [
    "Chromium"
  ],
  "homepage": "https://www.chromium.org/Home",
  "url": "https://download-chromium.appspot.com/dl/Mac?type=snapshots",
  "appcast": null,
  "version": "latest",
  "sha256": "no_check",
  "artifacts": [
    [
      "chrome-mac/Chromium.app"
    ],
    {
      "trash": [
        "~/Library/Preferences/org.chromium.Chromium.plist",
        "~/Library/Caches/Chromium",
        "~/Library/Application Support/Chromium",
        "~/Library/Saved Application State/org.chromium.Chromium.savedState"
      ],
      "signal": [

      ]
    }
  ],
  "caveats": "",
  "depends_on": {
  },
  "conflicts_with": {
  },
  "container": null,
  "auto_updates": null
```

Reply:
Seems good 👍